### PR TITLE
test(create): add missing tests, add classic feature (FE-2348)

### PR DESCRIPTION
**Pull request by DlgSHi**
### Checklist

<del> - [ ] Release notes (Conventional Commits)  </del> 
<del> - [ ] Unit tests </del> 
 - [x] Cypress automation tests

<del> - [ ] Storybook added or updated </del> 
<del> - [ ] Theme support </del> 
<del> - [ ] Typescript `d.ts` file added or updated </del> 

### Additional context

add missing tests for Dialog fullscreen and for Create components

### Testing instructions

run `npm run test-cypress` and run separate feature files

### CODE REVIEW

File: cypress/features/regression/create.feature [hunk 1]
```diff
@@ -4,6 +4,18 @@ Feature: Create component
   Background: Open Content component page
```
Comment (KatarzynaQA) [inline cypress/features/regression/create.feature]:
```suggestion
  Background: Open Create component page
```

File: cypress/features/regression/create.feature [hunk 2]
```diff
@@ -4,6 +4,18 @@ Feature: Create component
...
     Given I open "Create" component page
 
+  @positive
+  Scenario: Verify inner content and colors of Create component
+    # When I open "Create" component page
```
Comment (KatarzynaQA) [inline cypress/features/regression/create.feature]: add a comment why you commented the step

File: cypress/features/regression/createClassic.feature [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Feature: Create component classic page
+  I want to change Create component properties for classic page
+
+  Background: Open Content component page classic
```
Comment (KatarzynaQA) [inline cypress/features/regression/createClassic.feature]:
```suggestion
  Background: Open Create component page classic
```

File: cypress/features/regression/dialogFullScreen.feature [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Feature: Dialog Full Screen component
```
Comment (KatarzynaQA) [inline cypress/features/regression/dialogFullScreen.feature]: Should we add test related to event in Action Tab ?

File: cypress/features/regression/create.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/create.feature]: fixed

File: cypress/features/regression/create.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/regression/create.feature]: updated

File: cypress/features/regression/createClassic.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/createClassic.feature]: updated

File: cypress/features/regression/dialogFullScreen.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/regression/dialogFullScreen.feature]: updated

File: cypress/features/regression/create.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/create.feature]: updated

File: cypress/features/regression/create.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/regression/create.feature]: updated

File: cypress/features/regression/createClassic.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/createClassic.feature]: updated

File: cypress/features/regression/dialogFullScreen.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/regression/dialogFullScreen.feature]: added

**Pull request merged**